src/app/shared/components/flavour-icon/flavour-icon.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+    selector: "app-flavour-icon",
+    templateUrl: "./flavour-icon.component.html",
+    styleUrls: ["./flavour-icon.component.scss"]
+})
+export class FlavourIconComponent {
+    @Input() flavour: string = "";
+
+    get svgSource(): string {
+        return `assets/svgs/flavour-icons/${ this.filename }.svg`;
```

Comment:
This doesn't handle missing images, which cause a 404 error from angular-svg-icons. I wasn't sure how to handle this since Angular doesn't let you access `fs`, and adding an HTTP get request just to check if a file exists seemed like overkill, but it might be the only way, I'm not sure.